cmd/manager/main.go

```diff
@@ -91,6 +91,7 @@ func main() {
 	flag.StringVar(&cfg.Namespace, "namespace", cfg.Namespace, "Namespace where the CSIAddons pod is deployed")
 	flag.BoolVar(&enableAdmissionWebhooks, "enable-admission-webhooks", false, "[DEPRECATED] Enable the admission webhooks")
 	flag.BoolVar(&showVersion, "version", false, "Print Version details")
+	flag.StringVar(&cfg.SchedulePrecedence, "schedule-precedence", "", "The order of precedence in which schedule of reclaimspace and keyrotation is considered. Possible values are sc-only")
```

Comment:
can we have a validation for this one to ensure that only user is passing expected values?

Reply:
In case of invalid values, we fall back to the default precedence (ignoring it). The new precedence is used only when the value is `sc-only`. You want to exit in case of invalid value (we do that in case of configmap)?

Reply:
we need to validate and exit in the main.go or from where ever we are reading the value from the configmap, if we don't have the logs we will not get to know why it got skipped

Reply:
Done